packages/component/src/Styles/useInjectCustomProperties.ts

```diff
@@ -0,0 +1,58 @@
+import { hooks } from 'botframework-webchat-api';
+import { makeCreateStyles } from 'botframework-webchat-styles';
+import { useMemo } from 'react';
+import CustomPropertyNames from './CustomPropertyNames';
+import useInjectStyles from '../hooks/internal/useInjectStyles';
+import useNonce from '../hooks/internal/useNonce';
+
+const { useStyleOptions } = hooks;
+
+const webchatCustomPropertiesClass = 'webchat__css-custom-properties';
+
+export default function useInjectCustomProperties() {
```

Comment:
This seems like a private function that is only use by `bf-wc-component/Composer`.

Can we call it `useCustomPropertiesClassName(): readonly [string]`?

Resonating with my question above... should this put inside `bf-wc-styles` package?

Reply:
I don't want styles to depend on api. I think api -> component relationship is better meaning simpler hence left it here.